clients/launcher/pubspec.yaml

```diff
@@ -44,6 +44,7 @@ dependencies:
   crypto: ^3.0.6
   convert: ^3.1.1
   url_launcher: ^6.3.1
+  pointycastle: ^3.7.3
```

Comment:
tip: the AI is rarely up-to-date on the latest version. when you add a new package, check the latest version and use that instead.